api/router.go

```diff
@@ -184,21 +202,6 @@ func (mh *mainHandler) handle(w http.ResponseWriter, r *http.Request) error {
 		return nil
 	}
 
-	// Check authentication.
-	apiRequest.AuthToken = authenticateRequest(lrw, r, handler, readMethod)
-	if apiRequest.AuthToken == nil {
-		// Authenticator already replied.
-		return nil
-	}
-
-	// Wait for the owning module to be ready.
-	if moduleHandler, ok := handler.(ModuleHandler); ok {
-		if !moduleIsReady(moduleHandler.BelongsTo()) {
-			http.Error(lrw, "The API endpoint is not ready yet. Please try again later.", http.StatusServiceUnavailable)
-			return nil
-		}
-	}
-
 	// Add security headers.
```

Comment:
Should we move the security headers before the CORS check (just behind logging), so that they are always set?